### ISSUE TITLE
Document the fluent way to declare routes and groups

### ISSUE THREAD

**Issue by vpratfr** [laravel/docs#3379]
Some of the fluent methods to declare routes are not mentionned anywhere in the documentation. For instance `uses` and all of them for controllers.

Is that intentional?

E.g. from the docs:

```
Route::get('user/profile', 'UserController@showProfile')->name('profile');
```

Could now be:

```
Route::get('user/profile')
    ->uses('UserController@showProfile')
    ->name('profile');
```

or

```
Route::group(['prefix' => 'admin'], function () {
   // ...
});
```

could be written:

```
Route::prefix('admin')->group(function () {
   // ...
});
```

**Comment by connectkushal** [comment]
Created a pr #3380

**Comment by connectkushal** [comment]
This was merged in #3382

@vpratfr @themsaid please close the issue :)

**Comment by vpratfr** [comment]
Thanks
